src/nvim/testdir/test_writefile.vim

```diff
@@ -113,3 +113,21 @@ func Test_writefile_autowrite_nowrite()
   bwipe!
   set noautowrite
 endfunc
+
+func Test_writefile_fails_conversion()
```

Comment:
in vim this goes after the `Test_writefile_fails_gently` test. We need to keep the test files aligned to make merges easier.